README.md

```diff
@@ -56,6 +56,63 @@ There are five entrypoints to the backup module.
 4. `backup.sh`: Creates a lock directory and runs a one-off backup, then removes lock directory.
  If a lock directory exists and there is a running process, exits without doing anything.
   If a lock file exists and no running process, removes the lock directory.
-4. `schedule.py`: Creates a cron job that runs backup.py every hour and logs to `./log`. Must be run as root.
+4. `schedule.py`: Creates a cron job that runs backup.sh every hour and logs to `./log`. Must be run as root.
 5. `restore.py`:  TODO
 
+# Setup
+These are the steps followed to set up starport on the annex:
+1.  Add a bb8 user and group
+
+```
+sudo useradd bb8 -U -d /var/lib/bb8
+```
+2.  Ad the `bb8` user to the `ssh` group
+```
+sudo adduser bb8 ssh
+```
+
+3. Become the `bb8` user and change to the home directory
+```
+sudo su bb8
+cd ~
+```
+
+4. Clone this
+```
+git clone https://github.com/vimc/bb8
+```
+
+5. Create a `.ssh` directory for the `bb8` user
+```
+mkdir .ssh
+```
+
+6. Create the starport directory as the `bb8` user
```

Comment:
all this user stuff makes me really glad we have gone with docker elsewhere!  But also makes me wonder if this should all be running as docker too so that we can do all the setup as just an image and then do the backup into a further volume?